packages/xo-web/src/common/intl/messages.js

```diff
@@ -725,6 +725,8 @@ const messages = {
   customizeFilters: 'Customize filters',
 
   // ----- VM actions ------
+  cantRemotelyCopy:
+    "Your current edition doesn't support remote copy. You have to upgrade to Premium edition for this feature",
```

Comment:
`Remote copy requires at least enterprise plan` in muted text with an info icon? because it's just an info for users